lib/chef/resource/execute.rb

```diff
@@ -23,10 +23,6 @@ class Chef
   class Resource
     class Execute < Chef::Resource
       resource_name :execute
-      provides :execute
```

Comment:
?

Reply:
I didn't felt removing this is making any difference and it's required as I compared this with other resources. Let me know if I am missing something here.

Reply:
it is technically redundant with resource_name, but I'd like to deprecate resource_name, so having the `provides :execute` line in here is more forwards-looking.

Reply:
@lamont-granquist Fixed.